libse/SubtitleFormats/UnknownSubtitle41.cs

```diff
@@ -68,7 +69,7 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
                     {
                         if (line.Length > 1 && line.Length < 11 && RegexTimeCodes.Match(line.TrimEnd('/')).Success)
                         {
-                            p.EndTime = DecodeTimeCode(line.TrimEnd('/').Split('.'));
+                            p.EndFrame = DecodeFrames(line.TrimEnd('/').Split('.'));

```

Comment:
This is a frame based **subtitle format*, time must be calculated basing on  **Frame**.